src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -25,6 +25,11 @@
 #include "tvgCommon.h"
 #include "tvgArray.h"
 
+#ifdef __clang__
```

Comment:
This can be commonly used. I'd like to ask you to put this in tvgCommon.h

Reply:
done